### PR TITLE
remove all references to the Pipeline class

### PR DESCRIPTION
**Pull request by tylerl0706**
I think I'll need to try this on a v3 VM... sadly.

But this removes all the Pipeline references so that we can easily transition to PowerShellStandard.

### CODE REVIEW

File: src/PowerShellEditorServices/Session/PowerShellContext.cs [hunk 1]
```diff
@@ -823,20 +823,11 @@ public Task ExecuteCommand(PSCommand psCommand)
...
-                }
-                else
-                {
-                    pipeline = runspace.CreatePipeline(scriptToExecute, false);
-                }
-
-                Collection<PSObject> results = pipeline.Invoke();
+                pwsh.Runspace = runspace;
+                Collection<PSObject> results = pwsh.AddScript(scriptToExecute)
+                    .Invoke(null, new PSInvocationSettings() { AddToHistory = false });
```
Comment (SeeminglyScience) [inline src/PowerShellEditorServices/Session/PowerShellContext.cs]: `AddToHistory` is `false` by default, so you should be able to just use `Invoke()` here.

Also, there's a generic version of `Invoke` that specifies return type, so you can use `Invoke<TResult>()` and ditch the manual conversion code below.

**Comment by tylerl0706** [comment]
Tested V3. It works 😄 WATCH works while debugging a local script. Anything else I should test?

@SeeminglyScience

**Comment by rkeithhill** [comment]
Try doing something like `Get-Process $pid | Stop-Process -Confirm` from the PSIC. Then press S for suspend and try some commands.  Exit the nested prompt and then abort stopping the process.  

You can also try `$host.EnterNestedPrompt()` from the PSIC.

**Comment by SeeminglyScience** [comment]
@rkeithhill I don't think any of that works currently.  I can't remember what suspend is implemented with but `$Host.EnterNestedPrompt` is only implemented in the PSRL PR.

@tylerl0706 I can't remember what else uses that API.  Maybe try drilling down to properties in the variables view? I think that uses it.  Maybe just follow the rabbit hole of references and see what might be affected.

**Comment by rkeithhill** [comment]
>  I can't remember what suspend is implemented with 

According to Bruce's book WPIA, the engine calls back into the host: `PSHost.EnterNestedPrompt()` to process any commands entered while the command is suspended. But I don't have the brand new edition, and the previous edition which I do have only covers PS thru v2.

**Comment by tylerl0706** [comment]
Ok somehow that last commit is causing v3 to crash when debugging...

**Comment by tylerl0706** [comment]
```
Pipeline not run because a pipeline is already running. Pipelines cannot be run concurrently.
At C:\Users\tyleonha\Desktop\foo.ps1:2 char:1
+ $asdf = 1
+ ~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvalidOperation

```

Gonna sleep on this... this latest change leaves me with the error above...

**Comment by SeeminglyScience** [comment]
Eh... that's the error I was dreading.  It's probably a race condition that the little bit of conversion code "fixed" by adding some time between execution.  It may be better to wait to do this until 2.0 since the PSRL changes overhaul command invocation.

**Comment by tylerl0706** [comment]
So sad 😔

**Pull request closed without merging**